zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -349,22 +358,30 @@ public Set<String> getResourceIds() {
     public String getTenantId() {
       return tenantId;
     }
+
+    public String getForbiddenErrorMessage() {
+      return hasAddedResourceIds
```

Comment:
:upside_down_face: Could also check that `resourceIds` is not just the wildcard. Perhaps a bit less brittle than the boolean change flag.